Detectors/ITSMFT/MFT/macros/rootlogon.C

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
why is this necessary? ROOT6 finds libraries automatically.

Reply:
It is just to load the macro SetIncludePath, I will remove the libraries loading.

Reply:
Why do I have this error if the libraries are automatically loaded:

run_sim_mft_C.so: undefined symbol: _ZN2o23MFT8DetectorC1Ev

Reply:
Good question. I just quoted what the ROOT team told me ... So just leave it for the moment.